grapher/core/Grapher.tsx

```diff
@@ -491,11 +491,12 @@ export class Grapher
         this._receiveLegacyData(json)
     }
 
-    @computed get isAdmin() {
+    @computed get isAdminContext(): boolean {
```

Comment:
What about something boring but descriptive like `get loadDataViaAPI()`?

And then copy/paste the notion comment above it for future ref?